Source/SpiderEye.Windows/WinFormsWebview.cs

```diff
@@ -88,9 +88,9 @@ public Task<string> ExecuteScriptAsync(string script)
             return webview.ExecuteScriptAsync(script);
         }
 
-        public string RegisterLocalDirectoryMapping(string directory)
+        public Uri RegisterLocalDirectoryMapping(string directory)
         {
-            var host = UriTools.GetRandomFileHost();
+            var host = UriTools.GetRandomFileHostUrl("https");
```

Comment:
This breaks the windows implementation, as the host name passed to `SetVirtualHostNameToFolderMapping` shouldn't include a scheme

Reply:
Fixed on the `SetVirtualHostNameToFolderMapping` calling site. Could you verify the fix on win?